e2e/ibazel.go

```diff
@@ -92,6 +92,12 @@ func (i *IBazelTester) RunWithAdditionalArgs(target string, additionalArgs []str
 	i.run(target, []string{}, additionalArgs)
 }
 
+func (i *IBazelTester) RunUnverifiedWithAdditionalArgs(target string, additionalArgs []string) {
```

Comment:
What does it mean to be unverified? Is there maybe a different word or can we add a description for what this is?

Reply:
Really it's "execute bazel run without bazel build". I suppose "RunUnbuilt" might be a more accurate name?